docker/images/outrigger/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:14.04
+MAINTAINER Rodney Gomes (rodney@jut.io)
+
+CMD outriggerd
+
```

Comment:
I kinda agree it seems odd -- but it's consistent with what other dockerfiles do (e.g. postgres) so I would do it.